XouDevSpec/Classes/features/AnimeScreen/Repo/AnimeJsonTopStruct.swift

```diff
@@ -0,0 +1,32 @@
+
+
+
+import Foundation
+
+public struct animeTopInfoStruct: Decodable {
+    public var request_hash: String = ""
+    public var request_cached: Bool = true
+    public var request_cache_expiry: Int? = 0
+    public var top = [TopStruct]()
+    
+    public init() {
+    }
+        
+}
+
+public struct TopStruct: Decodable {
```

Comment:
Including the word Struct is not too correct in my opinion, would rather name it TopAnime or anything really. Up to you though, just naming

Reply:
Yeah, well said. I will change it.